.travis.yml

```diff
@@ -20,9 +20,11 @@ after_success:
 
 jobs:
   include:
+    - stage: "Themes"
+      script:
+        - julia --project=test/themes -e 'using Pkg; Pkg.instantiate(); Pkg.develop(PackageSpec(path=pwd()))'
+        - julia --project=test/themes test/themes/themes.jl
     - stage: "Documentation"
-      julia: 1.0
-      os: linux
```

Comment:
Will this not deploy from multiple workers?

Reply:
Nope, build stages [by default "inherit the first value of the array that defines a matrix dimension"](https://docs.travis-ci.com/user/build-stages/#build-stages-and-build-matrix-expansion). So in our case `os: linux` and `julia: 1.0`.